ocaml/idl/datamodel_vm.ml

```diff
@@ -1628,6 +1628,46 @@ let operations =
         ]
     )
 
+let set_blocked_operations =
+  call ~name:"set_blocked_operations"
+    ~in_product_since:rel_orlando (* but updated 2024 *)
+    ~doc:
+      "Update list of operations which have been explicitly blocked and an \
+       error code"
+    ~params:
+      [
+        (Ref _vm, "self", "The VM")
+      ; (Map (operations, String), "value", "Blocked operations")
+      ]
+    ~allowed_roles:_R_VM_OP (* was unrestricted when auto-generated? *)
```

Comment:
These we `_R_VM_ADMIN`, owing to the default for the class set at the `create_obj` call.
```
~messages_default_allowed_roles:_R_VM_ADMIN
```
So that was a little stricter than VM_OP.